mne/viz/epochs.py

```diff
@@ -69,6 +70,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    show_times : array-like, shape (n_epochs,) | None
```

Comment:
resp_times is bad because it could be all kinds of events that are not responses (for example, you could align to response and plot stim onset).

times isn't bad, but possibly confusable with the times param we already have - in topomaps. In eeglab, there's a (hidden, but cool) option for a joint topoplot/erpimage. In the medium run, I hope to do joint plots for all plot functions. For that reason, IMO the times parameter should have as few meanings as possible.

plot_times and time_lags don't seem bad to me right now. I don't mind show_times either.
plot_times sounds like a boolean as much as show_times does though.

+0.2 for time_lags.